test/extensions/filters/http/ext_proc/utils.cc

```diff
@@ -0,0 +1,24 @@
+#include "test/extensions/filters/http/ext_proc/utils.h"
+
+#include "test/test_common/utility.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace ExternalProcessing {
+
+bool ExtProcTestUtility::headerProtosEqualIgnoreOrder(
```

Comment:
Suggestion if you want to get fancy in the future, this can become a `MATCHER` that is usable under a gMock `EXPECT_THAT`

Reply:
Yes, next PR or two I can try that.